src/wrapmime.rs

```diff
@@ -25,6 +27,134 @@ macro_rules! clist_append {
     };
 }
 
+/**************************************
+* mime parsing API
+**************************************/
+
+pub fn get_ct_subtype(mime: *mut Mailmime) -> Option<String> {
+    unsafe {
+        let ct: *mut mailmime_content = (*mime).mm_content_type;
+
+        if !ct.is_null() && !(*ct).ct_subtype.is_null() {
+            println!("ct_subtype: {}", to_string((*ct).ct_subtype));
+            Some(to_string((*ct).ct_subtype))
+        } else {
+            None
+        }
+    }
+}
+
+pub fn get_autocrypt_mime(
+    mime_undetermined: *mut Mailmime,
+) -> Result<(*mut Mailmime, *mut Mailmime), Error> {
+    /* return Result with two mime pointers:
```

Comment:
does clippy approve of this style of comments?  anyway, below you use `//` so better be consistent and use `//` here too imho